src/helix/impl/analyzer.clj

```diff
@@ -128,6 +130,8 @@ Example: ($ %s %s ...)"
      :else
      (let [hd (first form)]
        (->> (cond
+              ('#{quote} hd)
+              nil
```

Comment:
same here
```suggestion
              ;; ignore quoted forms, since they aren't executable code
              ('#{quote} hd)
              nil
```